src/inject/inject.js

```diff
@@ -9,10 +9,10 @@ chrome.extension.sendMessage({}, function() {
       clearInterval(readyStateCheckInterval);
       chrome.runtime.onMessage.addListener(routeCommand);
     }
-  }, 10);
+  }, 3);
 
   function routeCommand(request) {
-    card = $('.active-card');
+    card = $('.card-detail-window');
```

Comment:
`$('.card-detail-window') || $('.active-card')` didn't work — see https://github.com/bulkan/shortcuts-for-trello/pull/9/commits/43dd575d78a65d115ff8c62bd7cd3ce199af671b for fix.